client/coral-embed-stream/src/containers/Embed.js

```diff
@@ -84,6 +85,9 @@ export const withQuery = graphql(EMBED_QUERY, {
       hasComment: commentId !== '',
       excludeIgnored: Boolean(auth && auth.user && auth.user.id),
     },
+    reducer: (previousResult, action, variables) => {
+      return reduceEditCommentActionsToUpdateStreamQuery(previousResult, action, variables);
```

Comment:
So far we have all our logic in mutations. I'm not sure if we should have some of our logic that modifies queries in mutations and some in the query reducer. What do you think?